3.0-RELEASE-NOTES.md

```diff
@@ -5,3 +5,10 @@ always describe the impact on users and instructions for upgrading
 applications from 2.x to 3.0.
 
 See also https://github.com/strongloop/loopback/blob/master/3.0-DEVELOPING.md
+
+## Always use bluebird as promise library
+
+In version 3.0, we always use bluebird as our promise library instead of `global.Promise`. We consider Bluebird API a part of LoopBack API from now on, you are welcome to use any Bluebird-specific methods in your applications.
```

Comment:
Please keep line width at 80 characters max, it makes it easier to review diffs.